css/index.css

```diff
@@ -1 +1,396 @@
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed,
+figure, figcaption, footer, header, hgroup,
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure,
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+html {
+	font-size: 62.5%;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: .1rem;
+    margin-bottom: 1.5rem;
+}
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+
+.container {
+  max-width: 880px;
+	width: 100%;
+  /* TB LR */
+  margin: 0 auto;
+
+}
+
+
+
+/* header */
+header {
+  display: flex;
+  justify-content: space-between;
+}
+
+header nav {
+	display: flex;
+  flex-flow: row flex-start;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 1rem;
+  width: 70%
+}
+
+header nav a {
+  display: flex;
+  flex-flow: row wrap;
+  font-size: 1.6rem;
+	font-weight: bold;
+  color: silver;
+  text-decoration: none;
+}
+
+header nav a:hover {
+  text-decoration: underline;
+}
+
+header nav a:first-child {
+  /*margin-left: 0;*/
+}
+
+header nav a:last-child {
+  /*margin-right: 0;*/
+}
+
+header .logo {
+  display: flex;
+  flex-flow: row-reverse;
+  margin-top: 3rem;
+	margin-bottom: 3rem;
+ }
+
+/* cta */
+.cta {
+  display: flex;
+  margin-top: 5rem;
+  justify-content: flex-end;
+	align-content: center;
+	align-items: center;
+	width: 100%;
+
+}
+
+.cta .cta-text {
+  display: flex;
+	flex-direction: column;
+  font-size: 7.7rem;
+  text-align: center;
+  align-items: center;
+	padding-top: 3rem;
+  align-self: baseline;
+	width: 50%;
+}
+
+.cta .cta-text button {
+  display: flex;
+  justify-content: center;
+  background: white;
+  margin: 1rem;
+	padding: 0.7rem;
+  font-size: 1.7rem;
+  border: 1px solid black;
+  cursor: pointer;
+  width: 40%;
+}
+
+.cta .cta-text button:hover {
+  background: black;
+  color: white;
+}
+
+.cta img {
+  display: flex;
```

Comment:
`display: flex` is unnecessary here on one img; it needs to be on the parent. However, you could add a `max-width` constraint.